xml/ay_bigfile.xml

```diff
@@ -3450,6 +3450,55 @@ openssl x509 -noout -fingerprint -sha256
 </screen>
    </sect2>
 
+   <sect2 xml:id="ay-use-whole-disk">
+    <title>Using the Whole Disk</title>
+
+    <para>
+     &ay; allows to use a whole disk without creating any partition by setting
```

Comment:
hello @imobachgs! Thanks for another PR! \o/
I suggest rephrasing this sentence, something like " &ay; allows formatting a whole disk as a single partition by setting the..."